docs/DeveloperGuide.md

```diff
@@ -398,11 +398,16 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* * *`  | user                                       | remove a student's profile                        | keep track of only students that I teach                     |
 | `* * *`  | user                                       | edit the details of a student                     | keep track of up-to-date information                         |
 | `* * *`  | user                                       | add individual tuition sessions                   | keep track of my tuition sessions                            |
-| `* * *`  | user                                       | delete an individual tuition session              | update my tuition schedule                                   |
+| `* * *`  | busy user                                  | add recurring tuition sessions                    | save time by creating multiple sessions in one command       |
+| `* * *`  | user                                       | delete a particular tuition session               | update my tuition schedule                                   |
```

Comment:
delete a particular non-recurring tuition session?

Reply:
*delete a non-recurring tuition session

Reply:
Ok got it! Have updated accordingly.